.github/workflows/ci.yml

```
@@ -29,72 +28,67 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.8", "3.9", "3.10"]
-        backend: [pyqt5]
-        include:
-          - platform: ubuntu-latest
-            python-version: 3.7
-            backend: pyqt5
+        platform: [ubuntu-latest, macos-13, macos-latest, windows-latest]
+        python-version: ["3.9", "3.10"]
+        backend: [PyQt5]
 
     steps:
       - uses: actions/checkout@v4
-      - uses: conda-incubator/setup-miniconda@v2
+      - uses: conda-incubator/setup-miniconda@v3
         with:
           miniconda-version: "latest"
```

```suggestion
          miniforge-version: "latest"
```
Can try to force miniforge, which maybe is smarter?

---

did it in last commit